Patch initdb success message for minimal container images.
Derived from docker-library/postgres Docker Official Image packaging.
https://github.com/docker-library/postgres

diff --git a/src/bin/initdb/initdb.c b/src/bin/initdb/initdb.c
--- a/src/bin/initdb/initdb.c
+++ b/src/bin/initdb/initdb.c
@@ -3256,3 +3256,1 @@
-		printf(_("\nSuccess. You can now start the database server using:\n\n"
-				 "    %s\n\n"),
-			   start_db_cmd->data);
+		printf(_("\nSuccess.\n\n"));
